src/error.rs

```diff
@@ -141,8 +143,13 @@ pub enum Error {
 
     /// The certificate is not valid for the Extended Key Usage for which it is
     /// being validated.
+    #[deprecated(since = "0.103.2", note = "use RequiredEkuNotFoundContext instead")]
```

Comment:
Can you add a `Cargo.toml` commit to the branch that bumps the version to 0.103.2 ?